src/sentry/api/serializers/models/event.py

```diff
@@ -285,8 +280,34 @@ def serialize(self, obj, attrs, user):
                 'tags': tags_meta,
             },
         }
+        # Serialize attributes that are specific to different types of events.
+        if obj.get_event_type() == 'transaction':
+            d.update(self.serialize_transaction_attrs(attrs, obj))
+        else:
+            d.update(self.serialize_error_attrs(attrs, obj))
         return d
 
+    def serialize_transaction_attrs(self, attrs, obj):
```

Comment:
I guess this and serialize_error_attrs should be "private" 
__serialize_error_attrs